packages/web-runtime/src/components/ApplicationsMenu.vue

```diff
@@ -21,11 +21,17 @@
     >
       <div class="uk-grid-small uk-text-center" uk-grid>
         <div v-for="(n, nid) in menuItems" :key="`apps-menu-${nid}`" class="uk-width-1-3">
-          <a v-if="n.url" key="apps-menu-external-link" :target="n.target" :href="n.url">
+          <a
+            v-if="n.url"
+            key="apps-menu-external-link"
+            :target="n.target"
+            :href="n.url"
+            tabindex="0"
+          >
             <oc-icon :name="n.iconMaterial" size="xlarge" />
             <span class="uk-display-block" v-text="$gettext(n.title)" />
           </a>
-          <router-link v-else key="apps-menu-internal-link" :to="n.path">
+          <router-link v-else key="apps-menu-internal-link" :to="n.path" tabindex="0">
```

Comment:
`tabinde="0"` shouldn't strictly be necessary (tab navigation works in Chrome without it and doesn't work in Firefox/Safari either way)

Reply:
solved